test/claim-service.test.js

```diff
@@ -0,0 +1,73 @@
+
+describe('Test claim service', () => {
+  let claimService
+  let claimRepositoryMock
+  let minetypeRepositoryMock
+  let messageServiceMock
+
+  beforeEach(async () => {
+    claimRepositoryMock = require('../server/repository/claim-repository')
+    minetypeRepositoryMock = require('../server/repository/minetype-repository')
+    messageServiceMock = require('../server/services/message-service')
+    jest.mock('../server/repository/claim-repository')
+    jest.mock('../server/repository/minetype-repository')
+    jest.mock('../server/services/message-service')
+
+    claimService = require('../server/services/claim-service')
+  })
+
+  test('Claim service create works with new claim', async () => {
+    const claimRecord = {
+      claimId: 'MINE123',
+      propertyType: 'business',
+      accessible: false,
+      dateOfSubsidence: new Date(),
+      mineType: ['gold', 'iron']
+    }
+    await claimService.create(claimRecord)
+    return expect(claimRepositoryMock.create).toHaveBeenCalledTimes(1)

```

Comment:
We should [test the arguments](https://jestjs.io/docs/en/mock-functions#custom-matchers) passed to `claimRepositoryMock.create()`. e.g.
```
expect(claimRepositoryMock.create).lastCalledWith(claimRecord)
```

The same applies to all other tests which assert that a function has been called.